frontend/app/routes/_gcweb-app.update-info.tsx

```diff
@@ -1,15 +1,44 @@
-import { type ActionFunctionArgs, json, redirect } from '@remix-run/node';
+import { type ActionFunctionArgs, type LoaderFunctionArgs, json, redirect } from '@remix-run/node';
 import { Form, useActionData, useLoaderData } from '@remix-run/react';
 
 import { z } from 'zod';
 
-import { type UserInfo, getUserInfo, updateUserInfo } from '~/services/user-info-client';
+import { getUserId, getUserInfo, updateUserInfo } from '~/services/user-info-service';
+
+export async function action({ request }: ActionFunctionArgs) {
```

Comment:
Move `action` after `loader` to follow Remix's convention.

Reply:
Now you're just getting silly.
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/pull/82/files#diff-2bec7e9aba6317cf91db5121aa046967350d305eba20a20d32dfa45459862afbR8-R29)